convert2rhel/pkghandler.py

```diff
@@ -1144,52 +1111,147 @@ def clean_yum_metadata():
     loggerinst.info("Cached yum metadata cleaned successfully.")
 
 
-def _package_version_cmp(pkg_1, pkg_2):
-    """Compare the version key in a given package name.
+def compare_package_versions(version1, version2):
+    """Compare two EVR versions against each other, as well as
+    the name and architectures
 
-    Consider the following variables that will be passed to this function::
 
-        pkg_1 = 'kernel-core-0:4.18.0-240.10.1.el8_3.x86_64'
-        pkg_2 = 'kernel-core-0:4.18.0-239.0.0.el8_3.x86_64'
+    :param version1: The version to be compared.
+    :type version1: str
+    :param version2: The version to compare against.
+    :type version2: str
 
-    The output of this will be a tuple containing the package version in a
-    tuple::
+    :example:
 
-        result = _package_version_cmp(pkg_1, pkg_2)
-        print("Result is: %s" % result)
-        # Result is: -1
+        >>> match = compare_package_versions("kernel-core-5.14.10-300.fc35", "kernel-core-5.14.15-300.fc35")
+        >>> match # -1
```

Comment:
This is more like an "accept if you like" type of suggestion, but, I always felt that if you include a `print` statement in the example, it would make much more sense. 

```suggestion
        >>> print(match) # -1
```

Reply:
I'll add this thanks